CMakeLists.txt

```diff
@@ -19,41 +20,43 @@ if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
             STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 endif ()
 
+# Set C++17 unless specified otherwise.
 if (NOT CMAKE_CXX_STANDARD)
     set(CMAKE_CXX_STANDARD 17)
     message(STATUS "${PROJECT_NAME} using CMAKE_CXX_STANDARD: ${CMAKE_CXX_STANDARD}")
 endif ()
 
-# Set extra compile options
-set(_iex_extra_compile_options_docstring "Add extra compile options to the CXX targets")
-if (MSVC)
```

Comment:
Documentation should probably state that this library is only designed to be built on unix-like operating systems.